content/700-optimize/400-recommendations/200-queries-on-unindexed-columns.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Queries on unindexed columns'
+metaTitle: 'Optimize Recommendations: Queries on unindexed columns'
+metaDescription: "Learn about the recommendation provided by Optimize for queries on unindexed columns."
+tocDepth: 3
+toc: true
+---
+
+The following queries targeting the `User` model use a [`where` property](/orm/prisma-client/queries/filtering-and-sorting) to filter on columns that do not have indexes:
+
+```ts
+await prisma.user.findFirst({
+   where: {
+      name: "Marc"
+   }
+})
+
+await prisma.user.findFirst({
+   where: {
+      name: "Jon"
+   }
+})
+
+await prisma.user.count({
+   where: {
+      name: "Nikolas"
+   }
+})
+```
+
+## Why is this a problem?
+
+An index allows the database to retrieve data more quickly, similar to how an index in a book helps you locate information without reading every page.
+
+When using Prisma with a `where` property, if no indexes are defined for the relevant columns, the database may need to scan every row in the table (a *“full table scan”*) to find matches. This can be undesirable for several reasons:
+
+### User experience
+
+For large datasets, if the database must scan the entire table to find matching rows, users will experience longer waiting times.
+
+### Resource utilization
+
+- **High CPU usage:** Scanning large tables can significantly increase CPU usage, degrading overall system performance.
+- **Memory consumption:** More memory is required to process and store data during a full table scan.
+- **Disk I/O:** Full table scans increase disk input/output operations, potentially slowing down other database activities.
+
+:::warning
+
+While these issues might not appear in development due to smaller datasets, they can become *significant* problems in production, where datasets are typically much larger.
+
+:::
+
+## Recommendation
```

Comment:
YEa this is the part that they need to use optimize for. This was great up until here. Maybe honestly just remove this

Reply:
@mhessdev remove what?